client/common/client.c

```diff
@@ -208,6 +208,46 @@ static BOOL freerdp_client_settings_post_process(rdpSettings* settings)
 		settings->Fullscreen = TRUE;
 	}
 
+	/* deal with the smartcard / smartcard logon stuff */
+	if (settings->SmartcardEmulation)
```

Comment:
why is this only used with emulated smartcards?

Reply:
Well, I don't see what sense it makes in the real smartcard scenario (pin set up by `/pin:XXXX`).
Are you considering that the `/p:password` would be the pin when using real smartcard ?

Reply:
Well, differing behaviour with emulation from a user perspective is not really a good idea.
so either keep the old fallback to use the password as pin or remove it in all cases?

Reply:
Well for sure, we want to have smartcard emulation as a runtime option instead of build option. I propose to keep that setting for that purpose.

Reply:
Yes there is, pkinit args are different